### ISSUE TITLE
pinch gesture works but swipe gestures not working

### ISSUE THREAD

**Issue by diode002** [bulletmark/libinput-gestures#246]
i installed it and follow proper steps but theirs a problem, it only works for pinch and not for swipe.

`:~$ id

uid=1000(diode) gid=1000(diode) groups=1000(diode),4(adm),24(cdrom),27(sudo),30(dip),46(plugdev),107(input),121(lpadmin),130(sambashare),134(lxd)`

`:~$ ls -l /dev/input

total 0
drwxr-xr-x 2 root root      60 فروری  24 00:14 by-id
drwxr-xr-x 2 root root     120 فروری  24 00:14 by-path
crw-rw---- 1 root input 13, 64 فروری  24 00:14 event0
crw-rw---- 1 root input 13, 65 فروری  24 00:14 event1
crw-rw---- 1 root input 13, 74 فروری  24 00:14 event10
crw-rw---- 1 root input 13, 75 فروری  24 00:14 event11
crw-rw---- 1 root input 13, 76 فروری  24 00:14 event12
crw-rw---- 1 root input 13, 77 فروری  24 00:14 event13
crw-rw---- 1 root input 13, 78 فروری  24 00:14 event14
crw-rw---- 1 root input 13, 66 فروری  24 00:14 event2
crw-rw---- 1 root input 13, 67 فروری  24 00:14 event3
crw-rw---- 1 root input 13, 68 فروری  24 00:14 event4
crw-rw---- 1 root input 13, 69 فروری  24 00:14 event5
crw-rw---- 1 root input 13, 70 فروری  24 00:14 event6
crw-rw---- 1 root input 13, 71 فروری  24 00:14 event7
crw-rw---- 1 root input 13, 72 فروری  24 00:14 event8
crw-rw---- 1 root input 13, 73 فروری  24 00:14 event9
crw-rw---- 1 root input 13, 63 فروری  24 00:14 mice
crw-rw---- 1 root input 13, 32 فروری  24 00:14 mouse0
crw-rw---- 1 root input 13, 33 فروری  24 00:14 mouse1`

**Comment by bulletmark** [comment]
Please follow the instructions in the TROUBLESHOOTING section. In particular, note the **bolded sentence** in the opening paragraph and last step. Also, please edit this issue title and make it meaningful.

**Comment by diode002** [comment]
after going through troubleshooting.

`diode@diode-PORTEGE-Z30-A:~$ libinput-gestures -l
libinput-gestures: session budgie-desktop+x11 on Linux-5.3.0-18-generic-x86_64-with-Ubuntu-19.10-eoan, python 3.7.5, libinput 1.14.1
/usr/bin/libinput-gestures: hash 06b487b84e7d5b0fb569204145519a72
Gestures configured in ~/.config/libinput-gestures.conf:
swipe up         3 notify-send Gesture performed
pinch in         2 notify-send Gesture performed
libinput-gestures: device /dev/input/by-path/platform-i8042-serio-1-event-mouse(event5): AlpsPS/2 ALPS DualPoint TouchPad
`

**Comment by bulletmark** [comment]
I edited your comment above to fix the formatting. You say you went through the troubleshooting steps but what did or did not work?! As step 8 says:

1. In step 4 did you see the commands printed to the screen? Please quote output here.
2. In step 5 did you see GESTURE_SWIPE and GESTURE_PINCH commands reported? Please quote output here.

Also, did you confirm your configured commands are working as per step 7?

**Comment by qiujiangkun** [comment]
I got the same problem after updating.
Everytime I restart the input-gesture, I can pinch in or out several times without any problems.
However, once I swipe, at the first time it works but not for the following times, even for pinching.
```log
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: Wrong slot count (2), reducing to 1
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: touch 1 ended and began in in same frame.
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: touch 1 ended and began in in same frame.
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: touch 1 ended and began in in same frame.
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: touch 1 ended and began in in same frame.
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: touch 1 ended and began in in same frame.
libinput error: event7  - SynPS/2 Synaptics TouchPad: kernel bug: touch 1 ended and began in in same frame.
```

**Comment by bulletmark** [comment]
@qiujiangkun you don't have the issue here. You have issue #244 which exists in libinput 1.15.1 and 1.15.2. The original issue there affected everybody after the update to libinput 1.15.1 and that was fixed in 1.15.2. However, a few people with "SynPS/2 Synaptics TouchPad" like you still see a [different bug in 1.15.2](https://gitlab.freedesktop.org/libinput/libinput/issues/434) which causes those "touch 1 ended" messages and touchpad gestures to stop working. See [my comment](https://github.com/bulletmark/libinput-gestures/issues/244#issuecomment-590588686) and the [the libinput bug](https://gitlab.freedesktop.org/libinput/libinput/issues/434).

So @qiujiangkun, please address any further comments over on issue @244, not here. The OP has not responded to my questions here so I will soon close this bug.

**Comment by bulletmark** [comment]
No response from OP so closing ...
